layers/meta-resin-raspberrypi/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,18 @@
+RESIN_EXTERNAL_MMC = "1"
+RESIN_INTERNAL_MMC = "0"
+UBOOT_KCONFIG_SUPPORT = "1"
+inherit resin-u-boot
+
+DEPENDS += "coreutils-native"
```

Comment:
Why is this dependency needed?

Reply:
u-boot needed this to build. Which is quite strange that this isn't already in one of the inherit/included files.
I think one of the bash scripts required something in coreutils-native.
I'll double check this